bootiful-banners-service/src/main/java/com/example/BootifulBannersServiceApplication.java

```diff
@@ -63,6 +64,28 @@ public static void main(String[] args) {
             boolean invert = this.properties.isInvert();
 
             String banner = imageBanner.printBanner(maxWidth, aspectRatio, invert);
+            if(ansiOutput == true) {
```

Comment:
You're right. There is a probably a nicer way by using a ByteArrayResource with ResourceBanner. Not sure it will be less lines of code but it would look smarter.